<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoToolsClass</class>
 <widget class="QMainWindow" name="SoToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>722</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>So可视化分析工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="elfTab">
     <attribute name="title">
      <string>ELF头部</string>
     </attribute>
     <widget class="QLabel" name="flagLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>处理器标志(Flags)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="flagKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>550</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="shstrndxKeyEdit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>510</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="phentsizeLabel">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>160</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>程序头部表格大小(Size of program headers)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="osAbiLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>系统环境(OS/ABI)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="shstrndxLabel">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>480</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>节区头部表格与节区名称字符串表相关索引(Section header string table index)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="typeKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="magicEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="ehsizeKeyEdit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>110</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ehsizeLabel">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ELF头部大小(Size of this headers)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phentsizeKeyEdit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>190</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="osAbiKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="osAbiValueEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="abiVersionLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ABI版本(ABI Version)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="abiVersionKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="typeValueEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>300</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="phnumLabel">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>240</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>程序头部表格数目(Number of program headers)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phnumKeyEdit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>270</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="shentsizeLabel">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>320</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>节区头部表格大小(Size of section headers)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="machineKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>340</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="machineLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>处理器类型(Machine)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="magicLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>魔法值(Magic)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="entryPointKeyEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>420</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="classLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件类别(Class)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="phoffLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>程序头部表格偏移量(Start of program headers)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="dataLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>数据编码(Data)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dataKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="eiVersionLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>头部版本(Version)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="eiVersionKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="typeLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件类型(Type)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="versionLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件版本(Version)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="versionKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="entryPointLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>程序入口(Entry point address)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="classKeyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="classValueEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="phoffKeyEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>460</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="dataValueEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="shoffLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>节区头部表格偏移量(Start of section headers)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="shoffKeyEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>500</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="shentsizeKeyEdit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>350</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="shnumLabel">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>400</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>节区头部表格数目(Number of section headers)：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="shnumKeyEdit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>430</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="phTab">
     <attribute name="title">
      <string>程序头部</string>
     </attribute>
     <widget class="QTableWidget" name="phTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>1100</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="phinfoLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1091</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="shTab">
     <attribute name="title">
      <string>节区头部</string>
     </attribute>
     <widget class="QLabel" name="shinfoLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1091</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="shTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>1100</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="fileOpen"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileOpen">
   <property name="icon">
    <iconset resource="SoTools.qrc">
     <normaloff>:/SoTool/Resources/icon/file_open.ico</normaloff>:/SoTool/Resources/icon/file_open.ico</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>请选择so文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SoTools.qrc"/>
 </resources>
 <connections/>
</ui>
